### ISSUE TITLE
Autocomplete is broken

### ISSUE THREAD

**Issue by TejasQ** [contiamo/operational-ui#847]
We broke Autocomplete in #842. Its current status on `master` is it looks pretty but does nothing. It is not keyboard accessible. It is not mouse accessible. It does not work. We need to fix this.

[Before](http://deploy-preview-800--operational-ui.netlify.com/#!/Autocomplete) | [Now](http://deploy-preview-842--operational-ui.netlify.com/#!/Autocomplete)

The goal is to make it like before. 😄
